helpers/markdownDocumentsReader.ts

```diff
@@ -22,7 +26,18 @@ export function getAllPostDocuments(): PostDocumentWithoutContent[] {
   const slugs = fs.readdirSync(documentsDirectory);
   const docs = slugs
     .map((slug) => getPostDocumentBySlug(slug))
-    .filter((post: PostDocument) => post.draft === false)
+    .filter((post: PostDocument) => {
+      if (isDev) {
+        return true;
+      } else if (
+        isPostADraft(post) === false &&
+        isPostInTheFuture(post) === false
+      ) {
+        return true;
+      } else {
+        return false;
+      }
```

Comment:
```suggestion
      if (isDev) {
        return true;
      } else if (
        isPostADraft(post) === false &&
        isPostInTheFuture(post) === false
      ) {
        return true;
      } else {
        return false;
      }
```
if (isDev) {
        return true;
      }

      if (isPostADraft(post) || isPostInTheFuture(post)) {
        return false;
      }

      return true;